### ISSUE TITLE
CodeApprove elides single lines in diff

### ISSUE THREAD

**Issue by mtlynch** [codeapprove/feedback#28]
I ran into a situation where CodeApprove elided a single line of code:

![image](https://user-images.githubusercontent.com/7783288/182677965-7c2cc394-8f0b-4a17-9368-84d6c0b7dbbc.png)

This is a little silly because in the space it took to elide the line, it could have just rendered the line.

**Comment by samatcodeapprove** [comment]
Yeah that's definitely not great. RIght now CodeApprove gets the initial diff directly from GitHub and GH sometimes makes strange decisions about where chunk boundaries are. But I can probably add an enhancement on my side to detect very small chunks and automatically expand them without user interaction.
